2013-02-20  Jakub Jelinek  <jakub@redhat.com>

	PR bootstrap/56258
	* doc/invoke.texi (-fdump-rtl-pro_and_epilogue): Use @item
	instead of @itemx.

	* gnat-style.texi (@title): Remove @hfill.

--- gcc/ada/gnat-style.texi.jj	2012-08-10 12:57:33.000000000 +0200
+++ gcc/ada/gnat-style.texi	2013-02-20 13:06:03.042667300 +0100
@@ -42,7 +42,7 @@ Texts.  A copy of the license is include
 @titlepage
 @titlefont{GNAT Coding Style:}
 @sp 1
-@title @hfill A Guide for GNAT Developers
+@title A Guide for GNAT Developers
 @subtitle GNAT, The GNU Ada Compiler
 @versionsubtitle
 @author Ada Core Technologies, Inc.
--- gcc/doc/invoke.texi.jj	2013-01-31 22:57:22.000000000 +0100
+++ gcc/doc/invoke.texi	2013-02-20 13:06:47.516405739 +0100
@@ -5612,7 +5612,7 @@ Dump after the peephole pass.
 @opindex fdump-rtl-postreload
 Dump after post-reload optimizations.
 
-@itemx -fdump-rtl-pro_and_epilogue
+@item -fdump-rtl-pro_and_epilogue
 @opindex fdump-rtl-pro_and_epilogue
 Dump after generating the function prologues and epilogues.
 
